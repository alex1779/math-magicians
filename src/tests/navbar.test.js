import renderer from 'react-test-renderer';
import React from 'react';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter
import Navbar from '../components/Navbar';

test('Confirm that the navbar is displayed correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Navbar />
    </MemoryRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
