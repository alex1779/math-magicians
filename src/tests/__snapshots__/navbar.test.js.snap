// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm that the navbar is displayed correctly 1`] = `
<nav
  className="nav"
>
  <h1
    className="h1"
  >
    Math Magicians
  </h1>
  <ul
    className="ul"
  >
    <li
      className="li"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      |
    </li>
    <li
      className="li"
    >
      <a
        className="navLink"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      |
    </li>
    <li
      className="li"
    >
      <a
        className="navLink"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
